packages/extension-chakra-storefront/src/components/skip-nav/index.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {Box, Link} from '@chakra-ui/react'
+
+interface SkipNavLinkProps {
+    children: React.ReactNode
+    zIndex?: string | number
+    href?: string
+}
+
+interface SkipNavContentProps {
+    children: React.ReactNode
+    style?: React.CSSProperties
+    id?: string
+}
+
+/**
+ * SkipNavLink component provides a skip link for keyboard navigation
+ * with initial state screen reader accessible but visually hidden
+ */
+export const SkipNavLink: React.FC<SkipNavLinkProps> = ({
+    children,
+    zIndex = 'skipLink',
+    href = '#skip-to-content',
+    ...props
+}) => {
+    return (
+        <Link
+            href={href}
+            position="absolute"
+            zIndex={zIndex}
+            css={{
+                position: 'absolute',
```

Comment:
Should we create theme file for this component?

Reply:
Done. I moved the styles to a theme.